notebooks/file_calibration.ipynb

```diff
@@ -61,6 +61,7 @@
     "from hera_notebook_templates.data import DATA_PATH as HNBT_DATA\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.    DO_CROSS_POL_CAL = os.environ.get("DO_CROSS_POL_CAL", "FALSE").upper() == "TRUE"
</pre><p>Since we use "DO" to mean something specific (the do scripts) it might be less confusing to just call this "CALIBRATE_CROSS_POLS"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Changed this to `CALIBRATE_CROSS_POLS`